test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -130,6 +131,27 @@ public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
         pipelineWatcher = new PipelineWatcher(containerComposer);
     }
     
```

Comment:
Could we add a new cleanUp method to combine `cleanUpPipelineJobs`, `createProxyDatabase` and `cleanUpDataSource` together to simplify invocation?